src/winetricks

```diff
@@ -15936,7 +15935,9 @@ load_wmp10()
 
     # Crashes on exit, but otherwise ok; see https://bugs.winehq.org/show_bug.cgi?id=12633
     w_try_cd "$W_CACHE/$W_PACKAGE"
-    w_try "$WINE" MP10Setup.exe $W_UNATTENDED_SLASH_Q
+    w_try cabextract -d "$W_TMP" ./MP10Setup.exe
+    w_try cd "$W_TMP"
```

Comment:
Please use `w_try_cd` here (shellcheck used to complain, not sure why it's not now, but I'd rather keep it consistent).